string bagPath
bool append
time startTime
time stopTime
---
